packages/mui-material/src/Select/Select.test.js

```diff
@@ -1,22 +1,22 @@
-import * as React from 'react';
+import Divider from '@mui/material/Divider';
```

Comment:
Please revert the change of import order. It's not related to this PR. We usually import `react` first, then external packages, and then @mui-scoped package.